### PR TITLE
[ch28750] Added a 404 & 500 html page

### PR DESCRIPTION
**Pull request by emaciupe**
[ch28750]

### CODE REVIEW

File: django_api/etools_prp/templates/http_codes_base.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <title>"UNICEF Partner Reporting Portal"</title>
+    <meta name="theme-color" content="#2E3AA1">
+
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+        // this is for the staging site only
+        ga('create', 'UA-70082239-1', 'auto');
```
Comment (domdinicola) [inline django_api/etools_prp/templates/http_codes_base.html:20]: is this the same ID as eTools???

Comment (emaciupe) [reply django_api/etools_prp/templates/http_codes_base.html:20]: @domdinicola ah yes, I knew I forgot to ask something here, this is a copy-paste, do we need GA for PRP? or remove it?

Comment (domdinicola) [reply django_api/etools_prp/templates/http_codes_base.html:20]: @robertavram  @ntrncic I think we should remove this (are we using matomo?)
maybe we will need to remove it also from eTools

Comment (domdinicola) [reply django_api/etools_prp/templates/http_codes_base.html:20]: @emaciupe please remove them and use matomo

Comment (emaciupe) [reply django_api/etools_prp/templates/http_codes_base.html]: resolved!

**Comment by emaciupe** [comment]
@ntrncic @domdinicola Requires to configure the matomo settings for backend.

**Comment by domdinicola** [comment]
@emaciupe do you think is possible to track informations about the user as well?

**Comment by emaciupe** [comment]
> @emaciupe do you think is possible to track informations about the user as well?

@domdinicola I've added tracking for the user id. 

> @emaciupe do you think is possible to track informations about the user as well?

@domdinicola yes, I've added user id tracking, see the screenshot on [Card #28750](https://app.shortcut.com/unicefetools/story/28750/add-404-page-similar-to-etools)

**Pull request merged**